config/internal/mcad/deployment.yaml.tmpl

```diff
@@ -25,7 +25,7 @@ spec:
           envFrom:
             - configMapRef:
                 name: mcad-{{.Name}}-config
-          image: 'quay.io/project-codeflare/mcad-controller:release-v1.29.57'
+          image: 'quay.io/project-codeflare/mcad-controller:release-v1.31.0'
```

Comment:
This has to be update here for the MCAD CRDs: 

https://github.com/project-codeflare/codeflare-operator/blob/a71303b7d9a2a59eb8310dbbaa351e2b43af256c/config/crd/mcad/kustomization.yaml#L2

Reply:
Updated. Good catch!